lib/themes/dosomething/paraneue_dosomething/templates/campaign/node--campaign.tpl.php

```diff
@@ -173,8 +173,10 @@
                     <li><a href="#" data-modal-href="#modal-shipment-form"><?php print $shipment_form_link; ?></a></li>
                   <?php endif; ?>
 
-                  <?php if (array_key_exists('enable_voter_registration', $campaign->variables) && $campaign->variables['enable_voter_registration']) : ?>
-                    <li><a href="#" data-modal-href="#modal-voter-registration">Register to Vote</a></li>
+                  <?php if (dosomething_campaign_feature_on($campaign, 'enable_voter_registration')) : ?>
```

Comment:
the fact that this feature is enabled should be stored in the `vars` array so you don't have to check if it's enabled a bunch of times in the theme